cli.js

```diff
@@ -31,7 +31,7 @@ yargs(process.argv.slice(2))
     format: {
       alias: 'f',
       desc: "Command for Python code formatter. Set to '' to disable.",
-      default: 'python -m black',
+      default: './mach lint --fix',
```

Comment:
question: What happens if the user's path is wrong here and there is not `mach` file?

Reply:
As you found in the next comment, they should get an appropriate error. Commands that start with `.` are sought not just in the current working dir, but also in the repo root, so it should be pretty robust.